data/json/harvest.json

```diff
@@ -388,6 +388,17 @@
       { "drop": "bone_tainted", "type": "bone", "mass_ratio": 0.1 }
     ]
   },
+  {
+    "id": "zombie_kevlar",
+    "//": "zombies with kevlar infused into their skin",
+    "type": "harvest",
+    "entries": [
+      { "drop": "meat_tainted", "type": "flesh", "mass_ratio": 0.25 },
+      { "drop": "fat_tainted", "type": "flesh", "mass_ratio": 0.08 },
+      { "drop": "bone_tainted", "type": "bone", "mass_ratio": 0.1 },
+      { "drop": "kevlar_plate", "type": "material", "mass_ratio": 0.015 }
```

Comment:
This triggers a load error about an invalid harvest type.

Reply:
type cannot be material. i suggest either "flesh" or "skin" depending on if you want it to be butcherable from the skinning action or not.

Reply:
Ah, I didn't realize. I thought I would have to use "material" for "kevlar_plate."
I'll make the change to "skin." Thanks!